### DISCUSSION TITLE
Can it be output as EUR or an optional currency?

### DISCUSSION THREAD

**Discussion by nguyenhung1121990** [infracost/infracost#2427] [Ideas & Feature Requests]
Hi team,

The InfraCost is really great. Our project is currently running on West Europe region so that would be nice if we could have value in EUR or can I be used for any optional currency?

Thanks a lot.

**Comment by hugorut** [comment]
Hi @nguyenhung1121990, This is currently possible using the currency configuration settings. See https://www.infracost.io/docs/features/environment_variables/#infracost_currency for more info.
